src/set_theory/principal.lean

```diff
@@ -109,7 +109,7 @@ end
 
 theorem unbounded_principal (op : ordinal → ordinal → ordinal) :
   set.unbounded (<) {o | principal op o} :=
-λ o, ⟨_, principal_nfp_blsub₂ op o, (le_nfp_self _ o).not_lt⟩
+λ o, ⟨_, principal_nfp_blsub₂ op o, (self_le_nfp _ o).not_lt⟩
```

Comment:
While you're renaming it, why not simply `le_nfp`?

Reply:
Sure, I like that. I'll apply this same rename to `self_le_nfp_(b)family`.